chore: rework imports in the editor based on PR comments

gashcrumb opened the pull request:
Took some time to go through the integration editor and rework the imports so they're not relative, touches related modules used by the editor as well.

deeleman commented:
this update is golden. I do not seem to be able to see the breakage reported by @kahboom . It LGTM

gashcrumb commented:
Fixed now.  I had added an `index.ts` to the integrations module but that seemed to break lazy loading.  For now I took that out so this is ready to go.

kahboom commented:
@deeleman - I didn't see any breakage, was trying to undo my approval while @gashcrumb resolved an issue he reported on IRC where lazy loading breaks and that the 'Integration' link stopped working.
